src/DetailsView/details-view-body.tsx

```diff
@@ -100,14 +98,8 @@ export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
     }
 
     private renderNavBar(): JSX.Element {
-        const tabClosed = this.props.tabStoreData.isClosed;
-
-        if (tabClosed) {
-            return null;
-        }
-
         return (
-            <DetailsViewLeftNav
+            <FluentSideNav
```

Comment:
This modifies the existing flow not under the feature flag, does it not?

Reply:
Yes, but right now inside the FluentSideNav it is just rendering the DetailsViewLeftNav.